cartridge/issues.lua

```diff
@@ -147,9 +154,9 @@ local function list_on_instance()
                 slab_info.quota_used_ratio
             ),
         })
-    elseif items_used_ratio > vars.limits.fragmentation_threshold_warning
-    and arena_used_ratio > vars.limits.fragmentation_threshold_critical
-    and quota_used_ratio > vars.limits.fragmentation_threshold_critical
+    elseif items_used_ratio > limits.fragmentation_threshold_warning
+    and arena_used_ratio > limits.fragmentation_threshold_critical
+    and quota_used_ratio > limits.fragmentation_threshold_critical
```

Comment:
> items_used_ratio > limits.**fragmentation_threshold_warning**
    and arena_used_ratio > **limits.fragmentation_threshold_critical**
    and quota_used_ratio > **limits.fragmentation_threshold_critical**

warning, critical, critical. Should it be "warning" in all places?

Reply:
We decided to use thresholds as is in documentation